tests/py/unit/test_method.py

```diff
@@ -0,0 +1,87 @@
+from mat3ra.mode import Method
+
+
+def test_can_be_created():
+    config = {"type": "pseudopotential", "subtype": "us"}
+    method = Method.create(config)
+    assert method.type == "pseudopotential"
+    assert method.subtype == "us"
+
+
+def test_type_property():
+    method = Method.create({"type": "pseudopotential", "subtype": "us"})
+    type_value = method.type
+
+    assert isinstance(type_value, str)
+    assert type_value == "pseudopotential"
+
+
+def test_subtype_property():
+    method = Method.create({"type": "pseudopotential", "subtype": "us"})
+    subtype_value = method.subtype
+
+    assert subtype_value is not None
+    assert subtype_value == "us"
+
+
+def test_data_property():
+    method = Method.create(
+        {
+            "type": "pseudopotential",
+            "subtype": "us",
+            "data": {"key": "value"},
+        }
+    )
+
+    data = method.data
+    assert isinstance(data, dict)
+    assert data["key"] == "value"
+
+
+def test_default_config():
+    config = Method.get_default_config()
+    assert "type" in config
+    assert "subtype" in config
+
+
+def test_set_data():
+    method = Method.create({"type": "pseudopotential", "subtype": "us"})
```

Comment:
Here and in other test files: `{"type": "pseudopotential", "subtype": "us"}` should be defined on top and reused like we do it in made